themes/default/content/case-studies/atlassian.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Increasing Developer Productivity at Bitbucket Cloud
```

Comment:
is it weird that this is the only page with a title that doesn't repeat the company name? at least from the list page it seems a bit out of place